presto-docs/src/main/sphinx/security/kerberos-configuration.fragment

```diff
@@ -0,0 +1,24 @@
+Kerberos Configuration
```

Comment:
Should this be titled `MIT Kerberos Configuration` since it seems to only apply to MIT Kerberos.